Sources/CoreCommands/Options.swift

```diff
@@ -473,7 +476,7 @@ extension AbsolutePath: ExpressibleByArgument {
     }
 }
 
-extension FingerprintCheckingMode: ExpressibleByArgument {
+extension WorkspaceConfiguration.CheckingMode: ExpressibleByArgument {
```

Comment:
Define a wrapper enum in `WorkspaceConfiguration` so we can avoid having `PackageFingerprint` and `PackageSigning` dependency in this module.